microsoft/testsuites/core/boot.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import SerialConsole
+from lisa.operating_system import CentOs, Redhat
+from lisa.tools import Reboot, Uname
+from lisa.util import SkippedException, TcpConnetionException
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite is to test VM working well after updating on VM and rebooting.
+    """,
+)
+class Boot(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. Skip testing if the distro is not redhat/centos type, RHEL added this test
```

Comment:
This information shouldn't be a step. We need to specify it's a RedHat contributed test case in the first line of description. Please also set the owner to "RedHat".

Reply:
under microsoft folder?

Reply:
Yes, we migrated it. They can move to their folder, if they want later.

Reply:
updated owner to RedHat